<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Упрощение текста</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="placeholderText">
     <string>Напишите сложный текст, который хотите упростить, в это окно и нажмите кнопку “Упростить”. Для больших текстов используйте кнопку “Выбрать файл”, чтобы выбрать текстовый файл формата txt. Переключатель &quot;Авто-копирование&quot; в активном положении автоматически копирует преобразованный текст в буфер обмена. Переключатель “Авто-сохранение” убирает окошко выбора сохранения файла, и вместо этого сохраняет преобразованный текст в папке входного файла с названием “*исходный-файл*-упр.txt”.</string>
    </property>
   </widget>
   <widget class="QLabel" name="ITlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Входной текст</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Упрощённый текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="OTlabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Преобразованный текст</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkCopy">
       <property name="text">
        <string>Авто-копирование</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSave">
       <property name="text">
        <string>Авто-сохранение</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSimplify">
       <property name="text">
        <string>Упростить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFile">
       <property name="text">
        <string>Выбрать файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="statusText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>891</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Строка состояния</string>
    </property>
   </widget>
   <widget class="QLabel" name="ITlabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>891</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Строка состояния</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ProgressLabel">
       <property name="text">
        <string>Прогресс</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="barText">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v/%m</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
